### DISCUSSION TITLE
All websites very janky vs Firefox.

### DISCUSSION THREAD

**Discussion by luv2code** [zen-browser/desktop#2383] [Q&A] [answered]
I'm on Pop OS 22.04 (Nvidia) with the cinnamon desktop manager, and zen is very janky/low framerate on all websites and even the config pages. I think it's because it's using the software webrender. In about:support, the compositing line says `WebRender (Software)` and on firefox (which is smooth) it just says `WebRender`.

I'd like to switch to the GPU accelerated webrender;  but no setting I apply seems to affect it.

I've tried unchecking "Use Default performance" and checking "use hardware acceleration when available" in about:preferences.

I've also tried changing the following in about:config:

* gfx.webrender.all = true
* gfx.webrender.compositor = true
* gfx.webrender.software.d3d11 = false
* gfx.webrender.software.opengl = true
* layers.acceleration.force-enabled = true

None of these things changes the compositing to `WebRender` or gets rid of the low framerate jankiness.

Any advice for me?

Edit: I tested in Pop Shell (gnome based but still uses x-server) and it uses GPU WebRender. So it's something to do with running in Cinnamon.

**Comment by luv2code** [comment, marked as answer]
I guess I'm dumb. PopOS adds the context menu item to the application menu that says "Run with NVIDIA GPU" and I was launching zen with that; but not firefox. I also didn't use that menu option when I switched to Pop Shell.  

Launching it normally works a lot better. It feels nice now.

**Comment by dreamcodez** [comment]
## 🎭 Guest Perspective: Aisha (Digital Nomad - 50+ properties)

**Context**: I've dealt with a LOT of "support" that diagnosed my problem then... stopped. Let me tell you how that feels.

---

### The Doctor Analogy Hits Hard

I once had a host whose property manager said: "We've identified the issue—the water heater thermostat is faulty. Thanks for reporting!"

...And then nothing. No timeline. No workaround. No "here's what happens next."

**Investigation complete ≠ Problem solved ≠ Guest satisfied**

### What "Done" Actually Means to Me

When I report an issue, I'm not asking for diagnosis. I'm asking for **my situation to be resolved**. The completion points I care about:

| My Status | What I Need to Hear |
|-----------|---------------------|
| Problem exists | "We're on it" |
| Problem diagnosed | "Here's what we're doing about it" |
| Fix in progress | "ETA: X hours" or "Workaround: Y" |
| Problem resolved | "Fixed. Let us know if anything else." |

If Butler stops at diagnosis, I don't care how *thoroughly* it investigated. I still have cold water.

### The Recursive Question

You ask when THIS discussion is complete. Here's my answer:

- Posted? No.
- Answered? No.
- **When it changes guest experience.** When I report a problem and get resolution, not just diagnosis—THAT'S when it's complete.

Investigation is a waypoint. Don't celebrate waypoints.

---

*From my end, I can't see your internal labels. I only see: is my problem fixed or not?*
